components/03-organisms/site-header/_site-header.scss

```diff
@@ -78,6 +79,22 @@ $site-header-layout: map.deep-get(tokens.$tokens, 'site-header-layout');
 
     [data-main-menu-state='closed'] & {
       max-height: 0;
+
+      // This animation is to remove hidden items from screen readers and
+      // keyboard navigation, while maintaining the slow "close" effect.
+      visibility: hidden;
```

Comment:
Interesting... I never noticed any issues here so I'd like to talk about this so I'm understanding this solution correctly. Nothing to change here before merging :)

Reply:
Yes! It is such a common issue with hamburger menus that I see. So the problem is that when the hamburger nav is closed, sighted users do not see the content (nav, util, nav, search,  etc). We want the same to hold true for keyboard- 
only and screen reader users. Otherwise, they will be tabbing through content that is positioned way off the page, not understanding the context. 

For example, on the SOM hamburger menu, there is a visibility:hidden on the header__menu wrapper. If I remove/uncheck that CSS property and tab passed the menu when it is closed, I will tab through all the menu options (that are floating off screen). 

[https://som.yale.edu/](https://som.yale.edu/)
<img width="388" alt="Screen Shot 2022-03-15 at 5 17 36 PM" src="https://user-images.githubusercontent.com/12991593/158474692-2afb6a93-49c8-44e8-99b1-0fbbb7b7748e.png">